packages/devtools_shared/lib/src/server/server_api.dart

```diff
@@ -36,6 +37,7 @@ class ServerApi {
     required ExtensionsManager extensionsManager,
     required DeeplinkManager deeplinkManager,
     ServerApi? api,
+    String? Function()? dtdUri,
```

Comment:
is this a function instead of a String so that it will always be up to date with the latest value on the server?

Reply:
you are correct :)